RELEASING.md

```diff
@@ -0,0 +1,8 @@
+# Sceptre release process
+
+1. Bump version in __init__.py & setup.cfg
+2. Update Changelog
+3. Create a PR for above changes
+4. `git tag -as vX.Y.Z`
+5. `git push origin vX.Y.Z` (CI/CD publishes to PyPi)
+6.  Email announcement of release
```

Comment:
This seems really vague. Who is being emailed? What would the content of this email be?

Reply:
Done.